lms/djangoapps/discussion/views.py

```diff
@@ -447,6 +453,7 @@ def _create_discussion_board_context(request, course_key, discussion_id=None, th
         'upgrade_link': check_and_get_upgrade_link(request, user, course.id),
         'upgrade_price': get_cosmetic_verified_display_price(course),
         # ENDTODO
+        'discussion_default_topic_id': _get_discussion_default_topic_id(course),
```

Comment:
@cahrens I added a comment for this. I used 'that contains "General"' instead of 'starts with'.